notifications/event-onclick.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Web Notifications Test: notification - onclick (basic)</title>
+<link rel="author" title="Apple Inc." href="http://www.apple.com/">
+<link rel="help" title="5 API" href="http://www.w3.org/TR/notifications/">
+<meta name="flags" content="">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+  <div id="log"></div>
+  <script>
+var t = async_test("the notification.onclick event can be invoked", {timeout: 50000});
+
+Notification.requestPermission(t.step_func(function(permission) {
+    assert_equals(permission, "granted", "notification permission");
+
+    var notification = new Notification("Please click on the notification.", {
+        body: "Click me to test clicking on notifications."
```

Comment:
Fixed in https://github.com/w3c/web-platform-tests/commit/a8d5f0411b8816c6c4aadf9896a47f6fbcf3431b